frontend/src/app/modules/user-card/user-card.component.sass

```diff
@@ -0,0 +1,166 @@
+@import '../../../styles/colors'
+
+*
+    box-sizing: border-box
+
+.card
+    padding: 23px 26px
+    max-width: 422px
+    display: flex
+    justify-content: center
+    align-items: center
+    background: $white
+    border-radius: 10px
+
+.content
+    display: flex
+    flex-direction: column
+    justify-content: space-between
+    width: 100%
+    height: 100%
+
+.up
+    display: flex
+    flex-direction: column
+    align-items: flex-start
+    border-bottom: 1px solid $gray-200
+    width: 100%
+    gap: 10px
+
+.left
+    justify-content: space-between
+    flex-direction: row
+    gap: 15px
+    display: flex
+
+.networks
+    display: flex
+    gap: 5px
+    height: 24px
+    justify-content: flex-end
+
+.picture
+    width: 56px
+    height: 56px
+    border-radius: 5px
+
+.info
+    display: flex
+    flex-direction: column
+    justify-content: space-between
+    gap: 5px
+
+.name
+    font-family: 'Barlow'
+    font-style: normal
+    font-weight: 700
+    font-size: 18px
+    color: $black
+
+.country
+    font-family: 'Barlow'
+    font-style: normal
+    font-weight: 500
+    font-size: 16px
+    display: flex
+    align-items: center
+    gap: 4px
+    color: $gray-300
+
+.social-icon
+    height: 24px
+    width: 24px
+    background-color: $turquoise-100
+    border-radius: 50%
+    display: flex
+    justify-content: center
+    align-items: center
+    padding: 2px
+    stroke-width: 1.5
+    stroke: $blue-200
+
+.fill-remaining-space
+    flex: auto
+
+.up_2
+    display: flex
+    justify-content: space-between
+    align-items: flex-start
+    gap: 11px
+    width: 100%
+
+.tags
+    display: flex
+    align-items: center
+    gap: 5px
+    width: 100%
+    padding-bottom: 13px
+
+.tag
+    display: flex
+    justify-content: center
+    align-items: center
+    background: $pink-100
+    border-radius: 50px
+    padding: 3px 8px
+    font-family: 'Barlow'
+    font-style: normal
+    font-weight: 400
+    font-size: 14px
+    line-height: 146%
+    color: $blue-200
+
+.bottom
+    display: flex
+    justify-content: space-between
+    align-items: center
+    gap: 20px
+    padding-top: 15px
+
+.progress-container
+    display: flex
+    align-items: center
+    gap: 10px
+    color: $gray-200
+
+.progress
+    display: flex
+    gap: 5px
+    font-family: 'Barlow'
+    font-style: normal
+    font-weight: 500
+    font-size: 16px
+
+.buttons
+    display: flex
+    align-items: center
+    gap: 10px
+
+.button
+    display: flex
+    justify-content: center
+    align-items: center
+    gap: 4px
+    font-family: 'Barlow'
+    font-style: normal
+    font-weight: 700
+    font-size: 14px
+    text-align: center
+    color: $blue-200
+    background: $white
+    border: 1px solid $blue-200
+    border-radius: 24px
+    padding: 8px 16px
+
+.spinner-container
```

Comment:
it would be better to create a separate shared component for this type of spinner and call it (round-progress-bar)